### PR TITLE
Fix translation files

### PR DESCRIPTION
**Pull request by mitchellcash**
Fixed English referencing in translation files for Bitcoin, Paycoin and
Peercoin

### CODE REVIEW

**Comment by mitchellcash** [comment]
Is there anything wrong with merging this PR? Changes are hopefully pretty self explanatory but any questions let me know.

If this can be merged I have further changes moving forward with translations and will be looking to assist the community by running a Transifex account with multiple translations and over time adding further languages and 100% complete translations. Just something I can do to help out more while letting the actual developers develop and not have to worry about this side of things. I have working local branches already :)

**Comment by PaycoinMaster** [comment]
Looks like a good pull. If a dev agrees we will merge this.

**Comment by IngCr3at1on** [comment]
All existing code references "Paycoin" rather than "PayCoin" (see bitcoinrpc.cpp for example) for consistency sake I'd say that's probably more appropriate. I've never really gotten the right info regarding trademark. @mgrav or @PaycoinMaster would know better on that one. 

Furthermore the copyright data should be the current Paycoin copyright (2014-2015) rather than the old Peercoin copyright (as far as that particular message is concerned, obviously having all the copyrights in the comments at the top of the files is still preferred).

The code itself looks fine, I'd want to compile it just for giggles before merging though.

**Comment by mitchellcash** [comment]
Thanks @IngCr3at1on 

I was unsure myself so I went with "PayCoin" because at the end of the day it was a 50/50 guess. I asked the question on talk.paycoin.com and no one knew and even the paycoin.com website uses both formats throughout the website and also the emails they send. If we could get some consensus on this it would be nice. In the meantime, I will change to "Paycoin".

Sorry can you expand on the copyright. I am changing the about dialog box in the qt to stop showing:
Copyright © 2014 **Paycoin** Developers
Copyright © 2011-2014 **Paycoin** Developers
Copyright © 2009-2014 Bitcoin Developers

and instead show

Copyright © 2014 **Paycoin** Developers
Copyright © 2011-2014 **Peercoin** Developers
Copyright © 2009-2014 Bitcoin Developers

I wasn't focusing on the copyright years at the moment just that we referenced Paycoin twice (because someone had just done a Find/Replace for Peercoin to Paycoin).

Let me know what you think and I will make whatever necessary changes.

**Comment by IngCr3at1on** [comment]
Yeah I mean if we want to change it to PayCoin personally I'm indifferent (definitely not something I would do on my own lol) but it needs to be changed everywhere.

In regards to the about dialog that's my mistake from reading just the commit and not the surrounding lines so that should be great.

**Comment by xmk3** [comment]
We need to leave it as Paycoin please.  PayCoin (with the capital C) is trademarked by an outside organization.  Thank you!  :-)

**Comment by mitchellcash** [comment]
@IngCr3at1on @xmk3 Thanks guys. Will make the necessary changes now

**Comment by mitchellcash** [comment]
@IngCr3at1on @xmk3 Thanks guys, this is now done.

**Comment by IngCr3at1on** [comment]
Looks good to me, I approve.

File: src/qt/locale/paycoin_de.ts [hunk 1]
```diff
@@ -259,7 +259,7 @@ Dieses Produkt enthält Software, die vom OpenSSL-Projekt zur Verwendung im Open
     </message>
     <message>
         <location filename="../askpassphrasedialog.cpp" line="107"/>
-        <source>WARNING: If you encrypt your wallet and lose your passphrase, you will &lt;b&gt;LOSE ALL OF YOUR PEERCOINS&lt;/b&gt;!
+        <source>WARNING: If you encrypt your wallet and lose your passphrase, you will &lt;b&gt;LOSE ALL OF YOUR PAYCOINS&lt;/b&gt;!
 Are you sure you wish to encrypt your wallet?</source>
         <translation>WARNUNG: Wenn Sie Ihre Wallet verschlüsseln und Ihre Passphrase verlieren, werden Sie &lt;b&gt;ALLE IHRE PEERCOIN VERLIEREN&lt;/b&gt;!
```
Comment (BoXHocK) [inline src/qt/locale/paycoin_de.ts]: Peercoin is mentioned instead of Paycoin here.

Comment (IngCr3at1on) [inline src/qt/locale/paycoin_de.ts]: Ahh in the translation itself good catch @BoXHocK

File: src/qt/locale/paycoin_ru.ts [hunk 2]
```diff
@@ -259,7 +259,7 @@ This product includes software developed by the OpenSSL Project for use in the O
     </message>
     <message>
         <location filename="../askpassphrasedialog.cpp" line="107"/>
-        <source>WARNING: If you encrypt your wallet and lose your passphrase, you will &lt;b&gt;LOSE ALL OF YOUR PEERCOINS&lt;/b&gt;!
+        <source>WARNING: If you encrypt your wallet and lose your passphrase, you will &lt;b&gt;LOSE ALL OF YOUR PAYCOINS&lt;/b&gt;!
 Are you sure you wish to encrypt your wallet?</source>
         <translation>ВНИМАНИЕ:  Если вы зашифровали свой кошелек и потеряли пароль, вы &lt;b&gt;ВСЕ ВАШИ PEERCOIN&lt;/b&gt;!
```
Comment (BoXHocK) [inline src/qt/locale/paycoin_ru.ts]: Here's one more I found. Will look through to check if there's more.

**Comment by mitchellcash** [comment]
Thanks @BoXHocK. I actually did notice this but was going to let it get swept up in the actual translating of the English version but I see what you're saying, I may as well fix it now. Will get it done right now.

**Comment by mitchellcash** [comment]
@BoXHocK @IngCr3at1on All done.

**Comment by IngCr3at1on** [comment]
works for me, as I said I approve :D

 @evanlucas @PaycoinMaster

**Pull request merged**